selenium-webdriver/chrome.d.ts

```diff
@@ -0,0 +1,418 @@
+import * as webdriver from "./index";
+import * as remote from "./remote";
+
+declare namespace chrome {
```

Comment:
did not find any uses for them in the API docs, moreover, you will have to add a types reference to it to work, since it automatically we will not pick up `selenum-webdriver\chrome`.